src/librustdoc/html/static/js/settings.js

```diff
@@ -1,4 +1,7 @@
 // Local js definitions:
+/* eslint-env es6 */
+/* eslint no-var: "error" */
+/* eslint prefer-const: "error" */
 /* global getSettingValue, getVirtualKey, onEachLazy, updateLocalStorage, updateSystemTheme */
 /* global addClass, removeClass */
 
```

Comment:
Should this and the other javascript files use `"use strict";`?

Reply:
It's mentioned in the issue already. I'll let this part of other contributors alongside adding more eslint checks.